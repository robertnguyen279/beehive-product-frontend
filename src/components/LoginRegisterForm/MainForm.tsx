import React from 'react';
import Login from './Login';
import Register from './Register';

const LoginRegisterForm = (): React.ReactElement => {
  const [isLogin, setLogin] = React.useState(true);

  return isLogin ? (
    <Login
      onChange={() => {
        setLogin(false);
      }}
    />
  ) : (
    <Register
      onChange={() => {
        setLogin(true);
      }}
    />
  );
};

export default LoginRegisterForm;
