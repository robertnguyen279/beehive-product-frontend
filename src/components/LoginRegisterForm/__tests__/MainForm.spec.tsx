import React from 'react';
import { shallow } from 'enzyme';

import MainForm from '../MainForm';
// const clickFn = jest.fn();

describe('MainForm', () => {
  const component = shallow(<MainForm />);

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
