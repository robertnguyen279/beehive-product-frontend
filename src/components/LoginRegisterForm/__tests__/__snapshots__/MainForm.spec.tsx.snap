// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainForm should render correctly 1`] = `
<Login
  onChange={[Function]}
/>
`;
